### DISCUSSION TITLE
How to use ChartAxisTicks/Callback?

### DISCUSSION THREAD

**Discussion by technyon** [Megabit/Blazorise#5554] [Q&A]
Hi,

I'm trying to use the setup custom ticks with the ticks callback option, but I can't figure out the syntax. I couldn't find anything in the documentation or otherwise, can someone give me an example how this is supposed to work?

Thanks!

**Comment by stsrki** [comment]
The syntax is as follows:

```
LineChartOptions lineChartOptions = new()
{
    AspectRatio = 1.5,
    Scales = new()
    {
        Y = new()
        {
            Title = new()
            {
                Display = true,
                Text = "Kelvins"
            },
            Ticks = new ChartAxisTicks
            {
                Callback = ( value, index, values ) => value / 1000 + "K",
                StepSize = 10000
            }
        }
    }
};
```

PS. We will also update the docs to make it easier.

**Comment by technyon** [comment]
Thanks

**Comment by technyon** [comment]
Hi,

The syntax is clear now, but I have another problem. I'm using the chart streaming plugin, and it seems that all ChartAxisTicks options are ignored, including the callback. Setting Display, ShowLabelBackdrop and so on has no effect at all. Is this expected?

P.S.: I'm setting ticks on the X-Axis.
